LibreHardwareMonitor/UI/MainForm.cs

```diff
@@ -854,22 +855,78 @@ private void TreeView_Click(object sender, EventArgs e)
+                            var confirmSensorselect = MessageBox.Show("Select the other sensor after clicking OK.", "New Manual Curve", MessageBoxButtons.OKCancel);
+                            if (confirmSensorselect == DialogResult.OK){
+                                // Listen for user click on sensor
+                                EventHandler selectorListener = null;
+                                selectorListener = (curveselect_sender, curveselect_e) => {
+                                    MouseEventArgs curveselect_m = curveselect_e as MouseEventArgs;
+                                    if (curveselect_m == null || curveselect_m.Button != MouseButtons.Left){
+                                        this.treeView.Click -= selectorListener;
+                                        return;
+                                    }
+                                    // Try find sensor user clicked
+                                    NodeControlInfo curveselect_info = treeView.GetNodeControlInfoAt(new Point(curveselect_m.X, curveselect_m.Y));
+                                    if (curveselect_info.Node != null){
+                                        SensorNode curveselect_node = curveselect_info.Node.Tag as SensorNode;
+                                        if (curveselect_node != null && curveselect_node.Sensor != null){
+                                            new SensorControlForm(node.Sensor, curveselect_node.Sensor, null).ShowDialog();
+                                            return;
+                                        }
+                                    }
+
+                                    var tryagainSensorselect = MessageBox.Show("Could not find sensor. Try again?", "Manual Curve", MessageBoxButtons.RetryCancel);
+                                    if (tryagainSensorselect != DialogResult.Retry)
+                                        this.treeView.Click -= selectorListener;
+
+                                };
+                                this.treeView.Click += selectorListener;
+                            }
+                        };
+                        curveItem.MenuItems.Add(newCurveItem);
+                        var softwareCurve = node.Sensor.Control.GetSoftwareCurve();
+                        if(softwareCurve != null)
+                        {                
+                            // edit curve
+                            MenuItem editCurveItem = new MenuItem("Edit");
+                            curveItem.MenuItems.Add(editCurveItem);
+                            editCurveItem.Click += delegate (object obj, EventArgs args)
+                            {
+                                new SensorControlForm(node.Sensor, softwareCurve.Sensor, softwareCurve.Points).ShowDialog(); 
+                            };
+
+                            if(control.ActualControlMode != ControlMode.SoftwareCurve)
```

Comment:
Add spaces before ( in if.

Reply:
Okay I think i've found all of these and fixed them.